test/server/listener_manager_impl_quic_only_test.cc

```diff
@@ -67,11 +69,18 @@ reuse_port: true
                    /* expected_value */ 1,
                    /* expected_num_calls */ 1);
 #endif
-
   expectSetsockopt(/* expected_sockopt_level */ SOL_SOCKET,
                    /* expected_sockopt_name */ SO_REUSEPORT,
                    /* expected_value */ 1,
                    /* expected_num_calls */ 1);
+#ifdef UDP_GRO
```

Comment:
Is this ifdef still needed?

Reply:
UDP_GRO is being used in the following expectSetsockOpt(...) call. 
Here ifdef is required to ensure that UDP_GRO is defined on the platform.